wallet/wallet.c

```diff
@@ -1770,8 +1769,11 @@ static bool wallet_stmt2htlc_in(struct channel *channel,
 	return ok;
 }
 
-static bool wallet_stmt2htlc_out(struct channel *channel,
-				struct db_stmt *stmt, struct htlc_out *out)
+/* Removes matching htlc from htlcs_in */
```

Comment:
Nit: this probably means `unconnected_htlcs_in`.

Reply:
I'm a bit skeptical about these conversion functions having side-effects.

Reply:
I agree.  It's something of a necessary evil though; the previous load-and-join split was pretty nasty, involving a temporary field which was bad too.

Reply:
Right, well at least we have documentation for it :-)